### PR TITLE
New post: design system in the making

### PR DESCRIPTION
**Pull request by BirgitBader**

### CODE REVIEW

File: _posts/2020/2020-01-07-design-system-in-the-making.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+date: 2020-01-07
+header_image: public/design-system-in-progress.jpg
+category: ux-and-design
+tags: ["UI", "design", "UI patterns", "design system"]
+authors: ["Birgit"]
+about_authors: ["bbader"]
+---
+
+A  while ago I posted here [why it makes sense to introduce a design system](/blog/ux-and-design/why-it-makes-sense-to-introduce-a-design-system/).
+Of course we want a design system here at ePages too!
```
Comment (ortana) [inline _posts/2020/2020-01-07-design-system-in-the-making.md]: The transition is a bit bumpy and I shortly stopped reading.

File: _posts/2020/2020-01-07-design-system-in-the-making.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+date: 2020-01-07
+header_image: public/design-system-in-progress.jpg
+category: ux-and-design
+tags: ["UI", "design", "UI patterns", "design system"]
+authors: ["Birgit"]
+about_authors: ["bbader"]
+---
+
+A  while ago I posted here [why it makes sense to introduce a design system](/blog/ux-and-design/why-it-makes-sense-to-introduce-a-design-system/).
+Of course we also want a design system here at ePages.
```
[the remark this answers is not in the document]
Comment (BirgitBader) [reply _posts/2020/2020-01-07-design-system-in-the-making.md]: No comma needed ;-)

File: _posts/2020/2020-01-07-design-system-in-the-making.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+So we're on it!
+How did we approach this?
+And what did we do so far?
+Continue reading... 😉
+
+## Realization
+
+If you stumble across the same challenges over and over again, you'll realize at some point that you need help.
+"Realization" came during our UI text sessions when we were confronted with new behavior, an element being used for something else than it was intended for, or "simply" disagreement about the target group.
```
[the remark this answers is not in the document]
Comment (BirgitBader) [reply _posts/2020/2020-01-07-design-system-in-the-making.md]: In German I'd say "... mit neuem Verhalten", in EN it's then "... new behavior"

File: _posts/2020/2020-01-07-design-system-in-the-making.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+We reached a pain point where we desperately longed for a recurring theme that supports us with designing our user interface.
+
+> A fault confessed is half redressed.
```
[the remark this answers is not in the document]
Comment (BirgitBader) [reply _posts/2020/2020-01-07-design-system-in-the-making.md]: yes, please :)

**Pull request merged**